cfgov/mega_menu/templates/modeladmin/mega_menu/menu/edit.html

```diff
@@ -0,0 +1,22 @@
+{% extends "modeladmin/edit.html" %}
+{% load i18n %}
+
+{% block footer %}
+    <footer>
+        <ul>
+            <li class="actions">
+                {% block form_actions %}
+                    {{ block.super }}
+                {% endblock %}
+            </li>
+            <li class="preview">
+                <a href="{{ view.preview_url }}"
+                    class="button icon icon-view"
+                    target="_blank" rel="noopener noreferrer"
```

Comment:
Maybe move `rel` to its own line like the other attributes. Also, maybe move the `>` to the last line of content, not on its own, but I don't really care. Also (trivial), we line up the attributes indents elsewhere (so `href` and `class` would be left-aligned), but there's an argument to be made for the indent you have too.